src/equiv.js

```diff
@@ -10,10 +10,11 @@ QUnit.equiv = (function() {
 	var parentsB = [];
 
 	function useStrictEquality( b, a ) {
-
```

Comment:
this blank line should be kept, all comments (even the jshint ones) should be preceded by a blank line.